### DISCUSSION TITLE
Does pg-promise 10.7.0 support PostgreSQL 14 version?

### DISCUSSION THREAD

**Discussion by dev-HardenLee** [vitaly-t/pg-promise#922] [Q&A]
Our company's API server currently uses pg-promise 10.7.0 with PostgreSQL version 11. However, we need to upgrade to PostgreSQL version 14. I am concerned about whether pg-promise 10.7.0 supports PostgreSQL 14.

I have read the official documentation but couldn't find the answer.

Could you provide the answer for me?

**Comment by vitaly-t** [comment]
Most likely it does. But what stops you from upgrading to the latest library?

**Reply by dev-HardenLee** [reply]
I think the usage depends on the version. I think if i upgrade library, Basic code is affected.

**Reply by vitaly-t** [reply]
[The changes](https://github.com/vitaly-t/pg-promise/releases/tag/11.0.0) are well known, no need to speculate.

**Reply by dev-HardenLee** [reply]
Thank you for your answer :)
